plugins/vli/fu-vli-usbhub-device.c

```diff
@@ -510,14 +510,11 @@ fu_vli_usbhub_device_guess_kind (FuVliUsbhubDevice *self, GError **error)
     			fu_vli_device_set_kind (FU_VLI_DEVICE (self), FU_VLI_DEVICE_KIND_VL819Q8);
                 break;
             default:
-                g_set_error (error,
-                        G_IO_ERROR,
-                        G_IO_ERROR_NOT_SUPPORTED,
-                        "hardware is not supported");
+                g_set_error (error, "Packet Type match failed: ");
```

Comment:
you need the domain and error code there

Reply:
I set tabstop to 8 so that should fix formatting... I hope. I'm not sure which domain and error codes should be placed here, or if its possible to skip g_set_error altogether?